setup.py

```diff
@@ -31,14 +31,13 @@
 Topic :: Software Development :: Libraries :: Python Modules
 """.splitlines())))  # noqa: E128
 
-SETUP_REQUIRES = ['setuptools_scm >= 3.0']
+SETUP_REQUIRES = ['setuptools >= 34.4', 'setuptools_scm >= 3.0']
```

Comment:
Good call, after moving to `importlib.metadata`/`importlib_metadata` we don't need a runtime dependency on `setuptools` anymore.  

Thanks for the PR!